guides/common/modules/proc_preparing-for-disaster-recovery-by-virtualizing-your-projectserver.adoc

```diff
@@ -0,0 +1,28 @@
+[id="preparing-for-disaster-recovery-by-virtualizing-your-{project-context}-server"]
+= Preparing for disaster recovery by virtualizing your {ProjectServer}
+
+Implement a reliable process for regularly taking VM snapshots of your virtualized {ProjectServer} and for backing up your snapshots for long-term storage.
+
+. Define a schedule for taking periodic snapshots of your virtualized {ProjectServer}.
+Consider your tolerance for potential data loss:
+Taking snapshots frequently will result in smaller amounts of data loss in case of a disaster.
+However, creating a snapshot takes time, and the snapshots also require storage space.
+. Define your snapshot retention policy.
+Consider how many snapshots you want to store: Regularly removing outdated snapshots helps optimize storage usage.
+. Using your hypervisor, schedule periodic snapshots of your {ProjectServer}.
+. Schedule periodic backups of the snapshots to prevent data loss in case of hypervisor failure.
++
+[NOTE]
+====
+While snapshots provide quick recovery points, backing up your snapshots gives you the ability for long-term storage and provides extra safety in case of a disaster on the side of your hypervisor.
+====
+
+// Is this a good place for an example? Something like a real-world scenario where a user schedules weekly snapshots, has automation in place to back up those snapshots, calculated that data loss will be XYZ so weekly snapshots are enough, removes snapshots older than 1 month...
+// Or maybe two or three examples, each suited for a different kind of environment? Example 1, example 2, example 3?
+// Is it possible to write down real-world examples like these?
```

Comment:
We might could come up with some, but I'd add those as a follow up as they might take more time to get right.

Reply:
Okay, let's stick with general steps and no example then. I won't even create a follow-up task/issue because we'll all probably have different priorities anyway.